packages/fx-core/templates/core/v3Migration/js.ts.app.yml

```diff
@@ -28,19 +35,82 @@ deploy:
       workingDirectory: tabs
```

Comment:
enable static website need to be like:
```
  - uses: azureStorage/enableStaticWebsite
    with:
      storageResourceId: ${{TAB_RESOURCE_ID}}
      indexPage: index.html
      errorPage: error.html
```

Reply:
This file is a template, the `$\{{ {{~placeholderMappings.[state.fx-resource-frontend-hosting.storageResourceId]~}} }}` will be replaced with real resource id's environment name based on user's bicep content.